src/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+import { NavLink } from 'react-router-dom';
+
+export const StyledNavbar = styled.nav`
+  text-align: center;
+  background: papayawhip;
+`;
+
+const Navbar = () => {
+  return (
+    <StyledNavbar>
+      <NavLink to="/">Home</NavLink>
```

Comment:
Is there any reason you are using `NavLink` over `Link`? Theres nothing wrong with this but the docs say `NavLink` is used when you want to add styling for that given path in the nav bar ( https://reacttraining.com/react-router/web/api/NavLink) so perhaps you could add some styling for when this path is active?